### PR TITLE
Add `Iterable.generate`, fix `List.generate`

### PR DESCRIPTION
**Pull request by wrbl606**
- added `Iterable.generate` support
- fixed `List.generate`

### CODE REVIEW

**Comment by wrbl606** [comment]
Resolves https://github.com/ethanblake4/dart_eval/issues/91

File: .vscode/settings.json
Comment (ethanblake4) [inline .vscode/settings.json:1]: Remove this file

**Comment by ethanblake4** [comment]
Looks good after you remove that settings file

File: .vscode/settings.json
Comment (wrbl606) [reply .vscode/settings.json:1]: Done!

**Comment by ethanblake4** [comment]
Merged, thank you! :)
FYI if you're planning to work on other issues you should use the new v0.7.0 branch as a base, it has a lot of changes.

**Pull request merged**